app/models/resource_request.rb

```diff
@@ -27,15 +33,9 @@ class ResourceRequest < ApplicationRecord
   # enqueue_derivativo_job ends up making requests to Derivativo which makes requests back to
```

Comment:
Just a note here that the old `enqueue_derivativo_job` still exists in this comment.